shell/platform/embedder/tests/embedder_unittests_gl.cc

```diff
@@ -3353,5 +3353,37 @@ TEST_F(EmbedderTest, MultipleDisplaysWithSameDisplayIdIsInvalid) {
   latch.Wait();
 }
 
+TEST_F(EmbedderTest, CanSetPixelGeometry) {
```

Comment:
While we are testing that the specification of the pixel geometry was noted, we are not actually testing that this caused updates to rendering. There are some pixel tests already setup up top that we can use to setup a test for the same.

Reply:
This won't actually change rendering just yet. 

AFAIK, this will only affect text rendering on supported platforms (Windows, Linux). And it won't do that until https://bugs.chromium.org/p/skia/issues/detail?id=10833 is fixed.

I think we will definitely want golden pixel tests for this once it is implemented, but there isn't enough in place yet for that. If you'd like to hold off on landing this until more is available for it we can.